tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,1005 @@
+
+          #end if
+
+        #end if
+
+        python '$__tool_directory__/astronomical_archives.py' '$output' '$output_csv' '$output_html' '$output_basic_html' '$output_error'
+      ]]>
+    </command>
+    <inputs>
+        <conditional name="archiveselection">
+            <param name="archive_type" type="select" label="Archive Selection">
+              <option value="registry">Query IVOA registry</option>
+              <option value="archive">Query specific IVOA archive</option>
+            </param>
+            <when value="registry">
+              <param name="keyword" type="text" label="Keyword" />
+              <param name="service_type" type="select" label="Service type">
+                <option value="TAP" selected="true">TAP: Tables</option>
+                <option value="SCS">SCS: Cone search</option>
+                <option value="SSA">SSA: Spectra</option>
+                <option value="SIA">SIA: Images</option>
+              </param>
+              <param name="wavebands" type="select" label="Wavebands">
+                <option value="all" selected="true">All</option>
+                <option value="radio">Radio</option>
+                <option value="millimeter">Millimeter</option>
+                <option value="infrared">Infrared</option>
+                <option value="optical">Optical</option>
+                <option value="uv">UV</option>
+                <option value="euv">EUV</option>
+                <option value="x-ray">X-ray</option>
+                <option value="gamma-ray">Gamma-ray</option>
+              </param>
+            </when>
+            <when value="archive">
+              <param name="archive" type="select" label="Archive">
+                <options from_file="astronomical_archives_gen.loc">
+                  <column name="value" index="2" />
+                  <column name="name" index="1" />
+                </options>
+              </param>
+            </when>
+        </conditional>
+        <section name="query_section" title="Query selection" expanded="false">
+          <conditional name="query_selection">
+            <param name="query_type" type="select" label="ADQL Query Selection">
+              <option value="none">No specific query (first n files from archive)</option>
+              <option value="obscore_query">IVOA obscore table query builder</option>
+              <option value="raw_query">Raw ADQL query builder</option>
+            </param>
+            <when value="obscore_query">
```

Comment:
Please check if this is what you used

```suggestion
            <when value="obscore_query">
            <!-- Fields defined https://www.ivoa.net/documents/ObsCore/20160224/WD-ObsCore-v1.1-20160224.pdf -- >
```

Reply:
I was kooking at it right now, it's not the same document but the tables content looks more or less the same